src/data/actions/comment.js

```diff
@@ -1,4 +1,4 @@
-import 'whatwg-fetch';
+import axios from 'axios';
```

Comment:
If this is something you folks want to keep consistent you could look into the [`eslint-plugin-import#import/order`](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#importorder-enforce-a-convention-in-module-import-order) rule.

Reply:
I would love to keep import order / newlines consistent in our frontends. Seems like this could definitely help with this. Thanks for sharing @jaebradley!